invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/details/index.html

```diff
@@ -0,0 +1,24 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2020 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+
+{%- set title = _("Search") + community.metadata.title -%}
+
+{%- block javascript %}
+    {{ super() }}
+    {{ webpack['invenio-communities-details-search.js'] }}
+{%- endblock %}
+
+{%- block page_body %}
+{{ super() }}
+<div data-invenio-search-config='{{
+  search_app_communities_records_config(app_id="rdm-search",
+  endpoint="/api/communities/"+community['uuid']+"/records") | tojson }}'></div>
```

Comment:
Maybe you can define the endpoint format in the `search_app_communities_records_config` i.e `/api/communities/{community_uuid}/records"` and inject the community uuid in the template afterwards...

Reply:
pseudocode:
```
currrent_rdm_records.resource.routes["community-records"]
```
try to check Werzeug `PathConverter` if something can be done to replace the `<pid_value>` param...